python/pyspark/sql/readwriter.py

```diff
@@ -804,8 +806,7 @@ def csv(self, path, mode=None, compression=None, sep=None, quote=None, escape=No
     def orc(self, path, mode=None, partitionBy=None, compression=None):
         """Saves the content of the :class:`DataFrame` in ORC format at the specified path.
 
-        ::Note: Currently ORC support is only available together with
-        :class:`HiveContext`.
+        .. note:: Currently ORC support is only available together with :class:`HiveContext`.
```

Comment:
We are deprecating HiveContext in 2.0, should we change this to say "Hive support"?